pagopa/pn-frontend
# PN-10641-fe-pf-pg-recapiti-messaggio-di-errore-inserimento-codice-otp-non-mostrato-al-secondo-tentativo

## Short description
This pull request improves error handling and subscription management by following the same schema of PG and the [FIGMA](https://www.figma.com/design/O6q8HrX3opuykOcj9idcKT/PF-Destinatari?node-id=274-72592#757588774).

## List of changes proposed in this pull request
- updated subscription and unsubscription of error events using `AppResponsePublisher` in `useEffect` hook.


## How to test
Start PF and navigate into contacts page and attempt to add a new contact using any available method, during the contact addition process, type an incorrect OTP code and repeat it two to three times consecutively, then observe the behavior of the application.
